src/react/views/admin/articles/edit.js

```diff
@@ -60,6 +65,10 @@ class ArticleEdit extends Component {
 }
 
 export default class ArticleEditPage extends Component {
+  getArticleContent() {
+    return this.articleEdit.contentEditor.serialize()['element-0'].value;
```

Comment:
We're using this module https://github.com/yabwe/medium-editor. 

`.serialize()`: returns a JSON object with elements contents

We only have one element and this is just the naming convention they use